File: src/main/java/play/pluv/playlist/infra/dto/OcrMusicResponse.java
package play.pluv.playlist.infra.dto;

import java.util.List;
import play.pluv.playlist.domain.PlayListMusic;

public record OcrMusicResponse(
    String artistNames, String songTitle
) {

  public PlayListMusic toPlayListMusic() {
    return new PlayListMusic(songTitle, List.of(artistNames), null, "");
  }
}

File: src/main/kotlin/play/pluv/playlist/infra/dto/OcrMusicResponse.kt
package play.pluv.playlist.infra.dto

import play.pluv.playlist.domain.PlayListMusic
import java.util.List

data class OcrMusicResponse(
    val artistNames: String, val songTitle: String
) {
    fun toPlayListMusic(): PlayListMusic {
        return PlayListMusic(
            title = songTitle,
            artistNames = List.of(artistNames),
            "",
            null,
        )
    }
}
